### PR TITLE
Update README.md - translations

### PR DESCRIPTION
**Pull request by jhenshall**
Updating with the translation workflow. Doing as PR because I've added quite a lot and would be great if you can have a check over @laurentS @Hagellach37 and see if my attempt at dev talk makes any sense.

One thing missing is for adding inline html for text e.g. `<strong>` that requires slight difference but it's too late to add now. 😴

### CODE REVIEW

**Comment by Hagellach37** [comment]
Hey @jhenshall for me this looks good and I could understand it.

There is one minor thing not touched and I'm not sure if we can handle it or not: There are some strings on the website that are set in javascript, e.g. the text in the legend of the map. I'm not sure how the workflow would work in such a case.

Other than that, nice work.

**Review by Hagellach37 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -12,24 +12,43 @@ The following instructions explain how to setup a local development environment
...
+### Adding new text
+- Add HTML tag as normal in the shtml page
+- Add the `data-i18n` attribute to the HTML tag with a unique id for the text
+	-  Must follow [page].[identifier] e.g. `data-i18n="homepage.hero-heading"`
+- Add the new `data-i18n` identifier to `locales/en.json`
+	- Must be nested properly within the correct page
+	- Please put in order of page (for easy reference, not technical requirement)
+
+### Translating text
+- Any changes made to `locales/en.json` are automatically sent to Transifex for translation.
```
Comment (laurentS) [inline README.md]: When the changes are pushed to github

**Review by laurentS — APPROVED** [review]
This is some seriously good documentation! Well done taking the time to write in all these details!

File: README.md (hunk 1 above)
Comment (jhenshall) [reply README.md]: Good spot, added.

**Comment by jhenshall** [comment]
Cheers @Hagellach37 @laurentS !  

For this one, any thoughts @laurentS? Maybe we should create an issue instead outside of this PR to look at later.
> There is one minor thing not touched and I'm not sure if we can handle it or not: There are some strings on the website that are set in javascript, e.g. the text in the legend of the map. I'm not sure how the workflow would work in such a case.

**Pull request merged**